mycore-cronjob/src/main/java/org/mycore/mcr/cronjob/MCRCronjobManager.java

```diff
@@ -119,11 +135,8 @@ private void scheduleNext(String jobID) {
         });
     }
 
-    public <T extends MCRCronjob> T getJob(String id) {
```

Comment:
Did you check if this method is used in MIR or other mycore projects?

Reply:
AFAIA it is not used. TBH, I didn't notice that this method is public, as I don't see its value in the public API. Should I keep it anyways?